app/controllers/concerns/export_service.rb

```diff
@@ -407,7 +407,8 @@ def export_work_metadata_as_csv(collection)
         'Pages Translated',
         'Pages Needing Review',
         'Pages Marked Blank',
-        'work_id'
+        'work_id',
+        'Contributors Users'
```

Comment:
Please relabel this to "Contributors" and move it before `work_id` in the column list.